FastlaneExample/fastlane/report.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="fastlane.lanes">
```

Comment:
I think this file should be added to the gitignore, if I remember correctly

Reply:
it is on the gitignore... strange